nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-cluster/src/main/java/org/apache/nifi/cluster/firewall/impl/FileBasedClusterNodeFirewall.java

```diff
@@ -56,7 +55,7 @@ public class FileBasedClusterNodeFirewall implements ClusterNodeFirewall {
 
```

Comment:
On Line 105 for unknown host we should avoid including the exception.  We know precisely why it was unknown and we're logging unknown host itself.  In previous behavior the stack would not print without debug so this will preserve that plus intent anyway.